GAME_DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,110 @@
+- A player has lost much health, and must risk what little life they have left to gather food. Note that the player cannot heal unless they have an accessory that consumes their energy. This is a situation the player has to face because of their own skill level.
+- A player summons an event that spawns waves of enemies to their base.
+- A player can claim a naturally generated structure by killing all monsters monsters in the area. Note that monsters do not respawn naturally.
+
+### Progression = Risk
+In order to progress in the game, the player must take risks. The player is otherwise free to stay where they are in terms of progression.
+
+**Examples:**
+- The deeper the player goes into the world's caves, the more valuables they can find, but consequentially, the more monsters they'll come across.
+- The player must defeat an enemy, boss, or event to obtain an item that helps them progress.
+- In a multiplayer world, a player could attack an enemy player to rise up in power.
+
+## Break the Cycle
+### No Dimensions
+Instead of creating seperate dimensions, we can fit these places physically into Cubyz' massive world to allow the player to come across them on their own.
+
+### No Teleportation
+To immerse the player and let them feel the sense of scale the world of Cubyz has, teleportation is not allowed, as the player will see less of the world that way.
+
+### No Automation
+Instead of staying in one place and farming everything, the player should have to take advantage of the infinite nature of the world to gain resources, meaning that a majority of resources are non-renewable.
+
+### Mobs Don't Respawn
+To prevent farming, mobs will not respawn naturally once killed. This will also make the player more aware that killing has long-term consequences in the area it occurs in.
+
+### No Passive Animals
+Instead of animals standing by to let you kill them, they will run from the player, attack back, and use defense mechanisms. This should make hunting and taming much more engaging.
+
+### No Unbreakable Tools
+In Minecraft, a tool is an investment you can make. You can enchant the tool, rename it, make it your own; there's an incentive to make it last forever with the use of mending. In Cubyz though, you are not creating an investment, but rather a tool to use until it breaks. If a player gets too attached to their tool, then they won't want to make other types of tools, and lower-tier materials will see less use as they will never be used in tool-making.
+
+### Avoid Clutter
+Inventories in Minecraft will often fill up with random items that the player does not want, making their inventory hard to manage. To mitigate this, try finding uses for existing items before adding new ones, and find ways to prevent items from finding their way into a player's inventory when they don't want it.
```

Comment:
```suggestion
Inventories will often fill up with random items that the player does not want, making their inventory hard to manage. To mitigate this, try finding uses for existing items before adding new ones, and find ways to prevent items from finding their way into a player's inventory when they don't want it.
```